internal/utils/determine_default_org.go

```diff
@@ -0,0 +1,48 @@
+package utils
+
+import (
+	"encoding/json"
+	"fmt"
+	"io"
+	"net/http"
+
+	"github.com/snyk/go-application-framework/internal/constants"
+)
+
+type selfDocument struct {
+	Data struct {
+		Attributes struct {
+			AvatarUrl         string `json:"avatar_url,omitempty"`
+			DefaultOrgContext string `json:"default_org_context,omitempty"`
+			Name              string `json:"name,omitempty"`
+			Username          string `json:"username,omitempty"`
+		} `json:"attributes,omitempty"`
+		Id   string `json:"id,omitempty"`
+		Type string `json:"type,omitempty"`
+	}
+}
+
+func GetDefaultOrgID(client *http.Client, apiUrl string) (orgID string, err error) {
+	url := apiUrl + "/rest/self?version=" + constants.SNYK_API_VERSION
```

Comment:
@PeterSchafer out of curiosity, can you link me to the rest api spec for this endpoint? We had lots of problems in determining user org, wonder if there's an endpoint finally :-)